src/components/form/FormInput.tsx

```diff
@@ -0,0 +1,34 @@
+import { Field } from 'formik';
+import { Fragment } from 'react';
+
+const FormInput = ({
```

Comment:
define an interface for Props, should extend FieldProps (which should exist in formik library)

Reply:
FieldProps extends: field, form, and meta.  Does provide any value?

Reply:
Having an issue with FieldProps.  Unable to make them conditionally and they are causing issues when importing data